Linking problem on RHEL 6

I'm updating the OpenBLAS package in Fedora and Fedora EPEL from 0.3.0 to 0.3.1. The packages are built with ```TARGET=CORE2 NO_AVX2=1 DYNAMIC_ARCH=1 DYNAMIC_OLDER=1```. The link phase fails with
```
dynamic.c: In function 'get_coretype':
dynamic.c:513: error: 'gotblas_SANDYBRIDGE' undeclared (first use in this function)
dynamic.c:513: error: (Each undeclared identifier is reported only once
dynamic.c:513: error: for each function it appears in.)
make[1]: *** [dynamic.o] Error 1
```
Full x86_64 build log at
https://kojipkgs.fedoraproject.org//work/tasks/8058/27978058/build.log

--- a/driver/others/dynamic.c
+++ b/driver/others/dynamic.c
@@ -510,7 +510,7 @@ static gotoblas_t *get_coretype(void){
 #ifndef NO_AVX2
 	  return &gotoblas_HASWELL;
 #else
-	  return &gotblas_SANDYBRIDGE;
+	  return &gotoblas_SANDYBRIDGE;
 #endif
 	  else
 	  return &gotoblas_NEHALEM;

